myRental/models/UsersRequests.js

```diff
@@ -0,0 +1,39 @@
+let bookshelf = require('../modules/bookshelfConfig.js').bookshelf;
+let bookshelfTest = require('../modules/bookshelfConfig.js').bookshelfTest;
+
+let User = bookshelf.Model.extend ({
+    tableName: "USERS",
+    duplicates: ["NAME", "EMAIL"],
+	requests: function() {
+		return this.belongsToMany(Request);
+	},
+});
+
+let UserTest = bookshelfTest.Model.extend ({
```

Comment:
What is this used for? If it's used for testing, should we just move it to it's own location? That way we wouldn't be mixing testing and production code.

Reply:
I see your point but I would like them close together because the data should be the same for both.